supergloo/examples/prometheus/prometheus-demo.yaml

```diff
@@ -0,0 +1,814 @@
+---
```

Comment:
So this file is pretty complicated and has a lot of stuff; do we need all this for the tutorial?